.github/workflows/cicd.yml

```diff
@@ -3,7 +3,7 @@ name: CI/CD
 on:
   push:
     branches:
-      - main
+      - master
```

Comment:
Could we instead change the `master` branch to be named `main`?
This would better align with the other repos, etc.